ncm-spma/src/main/pan/components/spma/apt/schema.pan

```diff
@@ -0,0 +1,16 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template components/spma/apt/schema;
+
+include 'components/spma/schema';
+
+type component_spma_apt = {
+    include structure_component
+    include component_spma_common
+    @{ Allow user defined (i.e. unmanaged) repositories to be present on the system }
```

Comment:
ah, the shortest way to for users to take over the machine. you sure you want this?